### ISSUE TITLE
LFS File Locks & 'check out' issues

### ISSUE THREAD

**Issue by alectron** [SRombauts/UE4GitPlugin#120]
Hello,
I try to use this for testing with Bitbucket.
But, there seem to be fundamental issues with file-locking.
i.e. when I do the following (from UE, but similar beahviour in UE if Git-Bash is used for locking).

Set a lock:
```
LogSourceControl: RunCommand: 'git lfs lock "Content/A/B.uasset"'
LogSourceControl: RunCommand(lfs lock):
Locked Content/A/B.uasset
```
which then outputs:
`SourceControl: Locked Content/A/B.uasset`
So that's a success. Seemingly.

After which the file has been locked succesfully in the git world, but - UE Source Control then errors with:
`LogSlate: Window 'Unable to Check Out From Source Control!' being destroyed`

And file save attempts trigger a warning that the file is writable, but " not checked Out From Source Control".

So what - to try to get to the bottom of the issue, and since a "check-out" is a perforce/sub-version concept, and not one that Git has... 
Is successfully setting a git lfs lock (which is achieved succesfully here in git), what else would be needed for UE Source Control and/or the Git Plugin (2.14 beta) to succesfully check-out a file from git/bitbucket...?
Cheers.

**Comment by AlKoulounil** [comment]
👍 

I have the same kind of problem here. git lfs locking and unlocking work fine on command line, but when doing it in UE, gives a error message.

For example, when basically clicking on check out for any asset:
![image](https://user-images.githubusercontent.com/16733285/78404433-6c5c7c00-75cc-11ea-9120-c9924a05ca58.png)

I get this error message:
![image](https://user-images.githubusercontent.com/16733285/78404468-7ed6b580-75cc-11ea-8acf-1838ab0d76e7.png)

Then, the asset is still correctly locked :
![image](https://user-images.githubusercontent.com/16733285/78404506-944bdf80-75cc-11ea-9d10-5206e05248d4.png)

Also, there is no option to unlock it, but I think this covered in another issue.

**Comment by AlKoulounil** [comment]
Well, actually , it looks also really similar to issues #116 and #109. Maybe this is all the same.

**Comment by AlKoulounil** [comment]
Well sorry for the disturbance, the solution in #109 worked for me : I had put my email instead of my git user name in the "uses Git LFS 2 file locking workflow" text box.

**Comment by stilobique** [comment]
Fix this issue with the previous comment, look #109 .
Why this field don't fill with the user name ?

**Comment by Angeloss** [comment]
I got this exact error on gitlab, username is the same as in gitlab, I also tried using the email as "User to lock files" but didn't work.

**Comment by AlKoulounil** [comment]
Yup, as I said, email does not work. I'd suggest you to check that you made no mistake with your user name (including lower / upper case), it's really what solved the issue in my case.

**Comment by perezd** [comment]
I get this issue as well on 4.25. Weirdly, it says it fails but then it actually works if I check `git lfs locks`. So its just misinterpreting the status or something.
